package.json

```diff
@@ -62,7 +62,10 @@
   },
   "pnpm": {
     "overrides": {
-      "@sveltejs/vite-plugin-svelte": "workspace:^"
+      "@sveltejs/vite-plugin-svelte": "workspace:^",
+      "semver@<7.5.2": ">=7.5.2",
+      "svelte": "$svelte",
```

Comment:
What is this magic $? pnpm using svelte stores now?

Reply:
Haha yeah pnpm has that documented at the bottom of [this section](https://pnpm.io/package_json#pnpmoverrides).